cfgov/v1/models/doc_detail_page.py

```diff
@@ -1,22 +1,11 @@
-import itertools
-
 from wagtail.wagtailcore.fields import StreamField
-from wagtail.wagtailadmin.edit_handlers import TabbedInterface, ObjectList, \
-    StreamFieldPanel
-from wagtail.wagtailcore import blocks
-from wagtail.wagtailcore.models import PAGE_TEMPLATE_VAR
 
-from .base import CFGOVPage
 from . import molecules
 from . import organisms
+from .learn_page import AbstractLearnPage
 
 
-class DocumentDetailPage(CFGOVPage):
-    header = StreamField([
-        ('text_introduction', molecules.TextIntroduction()),
-        ('item_introduction', organisms.ItemIntroduction()),
-    ], blank=True)
-
+class DocumentDetailPage(AbstractLearnPage):
```

Comment:
lets actually move this model to the learn_page.py seems unnecessary to have them separated now since their referring to doc details as learn pages